web/src/About.svelte

```diff
@@ -5,46 +5,88 @@
 
 <Modal bind:show={$showAbout}>
   <h1>The low-traffic neighbourhood (LTN) tool, v2</h1>
+
   <p>
-    This is an <b>experimental</b>
-    version of the
-    <a
-      href="https://a-b-street.github.io/docs/software/ltn/index.html"
-      target="_blank"
+    This is the second version of the A/B Street LTN tool, <a
+      href="https://github.com/a-b-street/ltn"
+      target="_blank">an open source project</a
+    >. This version, called the Connected Neighbourhoods Tool, has been funded
+    by
+    <a href="https://www.transport.gov.scot/" target="_blank"
+      >Transport Scotland</a
     >
-      A/B Street LTN tool
-    </a>
-    . Most parts of it do not work yet, and you should probably use the other version
-    of the tool instead.
+    and designed by
+    <a href="https://www.sustrans.org.uk" target="_blank">Sustrans Scotland</a>.
+    The team includes:
   </p>
-  <p>To use this tool, you need to:</p>
-  <ol>
-    <li>Choose your study area to analyze</li>
-    <li>Define one or more neighbourhood boundaries</li>
+  <ul>
+    <li>
+      Dustin Carlino, lead developer and director of <a
+        href="https://find-and-update.company-information.service.gov.uk/company/16217327"
+        target="_blank">A/B Street Ltd</a
+      >
+    </li>
     <li>
-      Optionally, fix the basemap data on existing modal filters, turn
-      restrictions, one-ways, etc
+      <a href="https://github.com/michaelkirk/" target="_blank">Michael Kirk</a
+      >, software developer
     </li>
     <li>
-      Create one or more proposals with new modal filters, and explore their
-      effects
+      Sustrans Scotland
+      <ul>
+        <li>
+          <a href="https://www.linkedin.com/in/congying-hu/" target="_blank"
+            >Congying (Cici) Hu</a
+          >, mobility planner
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/in/angus-calder-a69543206"
+            target="_blank">Angus Calder</a
+          >, senior mobility planner
+        </li>
+        <li>Michael Naysmith, senior mobility planner (PM)</li>
```

Comment:
A minor thing, but is it well understood what PM means in this context?

Reply:
Good point; Michael put this in the google doc directly, but expanding the acronym to be more clear